script/swap_and_bridge/example/e2e_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
```

Comment:
```suggestion
cd ../../../
```

Reply:
I usually run every script from the script folder, but I agree this is unclear. If I clarify that in the README would that be fine?

Reply:
All scripts inside `script` folder and its subfolders are currently executed within root `script` folder. See example:
https://github.com/LiskHQ/lisk-contracts/blob/development/script/paused/deployPausedClaim.sh#L8
Would be nice to use same pattern for scripts inside this PR.

Reply:
Currently all scripts inside `script` folder and its subfolders are executed within a folder where the script is located. See example:
https://github.com/LiskHQ/lisk-contracts/blob/development/script/paused/deployPausedClaim.sh#L8
Would be nice if you use this pattern also inside this PR.